packages/story-editor/src/components/panels/design/videoOptions/videoOptions.js

```diff
@@ -136,6 +140,20 @@ function VideoOptionsPanel({ selectedElements, pushUpdate }) {
       </Row>
       {canMute && (
         <>
+          {enableVideoVolume && (
+            <Row>
+              <NumericInput
+                value={volume}
+                onChange={onChangeVolume}
+                min={0}
```

Comment:
As per the ticket description, the min should be 0.1. Why? Because 0.0 technically means muted, but we mute videos separately using FFmpeg and the `muted` attribute.

Reply:
I force the min value to be elsewhere.